### ISSUE TITLE
Fails to execute jq query when the key starts with `@` symbol

### ISSUE THREAD

**Issue by Faroukhamadi** [ynqa/jnv#22]
The problem occurs when `@` symbol is present as a key in a json file, I get autocompletion when I press Tab but then it fails to get the value for the specified key.
![reproduction](https://github.com/ynqa/jnv/assets/36114668/57091900-c8b3-401c-a0b6-f653b96f5fdb)

**Comment by ynqa** [comment]
Easy repro:

```bash
$ echo '{ "@ynqa": "need double quotation for query" }' | jnv

❯❯ .@ynqa
Failed to execute jq query '.@ynqa'
  .
❯ .@ynqa
{
  "@ynqa": "need double quotation for query"
}
```

**Comment by ynqa** [comment]
@Faroukhamadi Thank you for the report. It seems that if there is an `@` present, it needs to be escaped with double quotes. This is something that hadn't been addressed, and will be fixed 👍

**Comment by ynqa** [comment]
@Faroukhamadi I just released v0.1.3, please check 🎉

**Comment by Faroukhamadi** [comment]
@ynqa Will do, thanks for the quick fix/response! 🎉🎉
